Pipfile

```diff
@@ -0,0 +1,15 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+aiohttp = "*"
+pytest = "*"
+sanic = "*"
+sanic-openapi = {path = "."}
+
+[packages]
+
+[requires]
+python_version = "3.5"
```

Comment:
I don't think put `Pipfile` into this project is a good choice because we do not limit or recommend use which approaches to join the developing. But this still a good reference, maybe put it to `README.md` or `CONTRIBUTING.md`(I'm working on it) would be very nice.

Reply:
I intentionally added only `Pipfile`, and not `Pipfile.lock`. I don't feel having `Pipfile` in the repo makes Pipenv the recommended dev tool (`Pipfile.lock` is a stronger statement), but it's your project, so of course i will remove it, no problem.
Anyway, i only created the Pipenv environment, because there are some dev dependencies that need to be in place (`aiohttp` and `pytest`) if i want to run the test suite.